packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -173,10 +173,6 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 		itExpects(
 			"Send ops fails for swept datastores in summarizing container loaded before sweep timeout",
 			[
-				{
-					eventName: "fluid:telemetry:Summarizer:Running:SweepReadyObject_Loaded",
```

Comment:
I'm having a little bit of mixed feelings about these logs going away. They are a pain to maintain, they're always popping up in tests everywhere.  But I also like having test coverage _somewhere_ to prevent regressions, so when we see these logs we can be confident we're interpreting them properly.

Might not be worth it, thinking out loud.

Reply:
I was never a fan of validation via the `itExpects` block. IMO, if we want to validate these events, we should make it part of the test itself.

Reply:
Yeah, I'll add a low-pri note to the follow-ups, to double-check if we have coverage using MockLogger anywhere and consider adding some.